src/components/checkboxes/_checkbox-macro.njk

```diff
@@ -1,12 +1,13 @@
 {% macro onsCheckbox(params) %}
     {% from "components/label/_macro.njk" import onsLabel %}
 
-    <span class="ons-checkbox {{ params.classes }}">
+    <span class="ons-checkbox{% if params.hideLabel is defined and params.hideLabel == true %} ons-checkbox--no-label{% endif %}{{ ' ' + params.classes if params.classes and params.classes is defined else '' }}">
         <input
             type="checkbox"
             id="{{ params.id }}"
-            class="ons-checkbox__input ons-js-checkbox {{ params.inputClasses }}"
+            class="ons-checkbox__input ons-js-checkbox{{ ' ' + params.inputClasses if params.inputClasses and params.inputClasses is defined else '' }}"
```

Comment:
The `' '` should be inside the `if` otherwise it would leave whitespace in the class chain.

Reply:
This change was from master, so I've raised #2239 to address this